### ISSUE TITLE
On Search page, remember criteria when switching tabs

### ISSUE THREAD

**Issue by poonen** [roed314/seminars#419]
On the Search page, it would be nice if the Subject, Topic, Institution, Language, and Access criteria were remembered when I switch from one tab to the next.

**Comment by poonen** [comment]
As of now, these are remembered only after clicking "Search...".  It would be nice if they were saved from the moment they are entered.

**Comment by roed314** [comment]
This is no longer relevant, and the More filter implements a version of this.

**Comment by poonen** [comment]
It would still be nice to have this for the search box on the Browse page, I think.  I'll create a separate issue for this.
